inc/classes/subscriber/Addons/Cloudflare/CloudflareSubscriber.php

```diff
@@ -196,6 +196,16 @@ public function auto_purge_by_url( $post, $purge_urls, $lang ) {
 		if ( is_wp_error( $cf_cache_everything ) || ! $cf_cache_everything ) {
 			return;
 		}
+		// Add home URL and feeds URLs to Cloudflare clean cache URLs list.
+		$home_url     = get_rocket_i18n_home_url( $lang );
+		$purge_urls[] = $home_url;
```

Comment:
As `$home_url` is not used elsewhere in this function, we can combine these 2 lines of code:

```php
$purge_urls[] = get_rocket_i18n_home_url( $lang );
```